python/shotgun_desktop/initialization/shotgun.py

```diff
@@ -16,6 +16,9 @@
 
 from . import constants
 from distutils.version import LooseVersion
+from ..logger import get_logger
+
+logger = get_logger("initialization.shotgun")
```

Comment:
With the small refactoring I've done of the logger, I it would certainly be easy to simply reparent everything under sgtk.desktop instead of tk-desktop and be forward compatible with 0.18. Good idea.